client/extensions/woocommerce/app/orders/orders-list.js

```diff
@@ -0,0 +1,180 @@
+		}
+	}
+
+	componentWillReceiveProps( newProps ) {
+		if ( newProps.currentPage !== this.props.currentPage || newProps.siteId !== this.props.siteId ) {
+			this.props.fetchOrders( newProps.siteId, newProps.currentPage );
+		}
+	}
+
+	getOrderStatus = ( status ) => {
+		const { translate } = this.props;
+		const classes = `orders__item-status is-${ status }`;
+		switch ( status ) {
+			case 'pending':
+				return <span className={ classes }>{ translate( 'Pending payment' ) }</span>;
+			case 'processing':
+				return <span className={ classes }>{ translate( 'Processing' ) }</span>;
+			case 'on-hold':
+				return <span className={ classes }>{ translate( 'On hold' ) }</span>;
+			case 'completed':
+				return <span className={ classes }>{ translate( 'Completed' ) }</span>;
+			case 'cancelled':
+				return <span className={ classes }>{ translate( 'Cancelled' ) }</span>;
+			case 'refunded':
+				return <span className={ classes }>{ translate( 'Refunded' ) }</span>;
+			case 'failed':
+				return <span className={ classes }>{ translate( 'Failed' ) }</span>;
+		}
+	}
+
+	renderOrderItems = ( order, i ) => {
+		const { translate, moment, siteId } = this.props;
+		return (
+			<TableRow key={ i }>
+				<TableItem className="orders__table-checkbox">
+					<FormInputCheckbox aria-label={ translate( 'Select order %(order)s', {
+						context: 'Label for checkbox',
+						args: {
+							order: order.number,
+						}
+					} ) } />
+				</TableItem>
+				<TableItem className="orders__table-name" isRowHeader>
+					<a className="orders__item-link" href={ `/store/order/${ siteId }/${ order.number }` }>#{ order.number }</a>
+					<span className="orders__item-name">
+						{ `${ order.billing.first_name } ${ order.billing.last_name }` }
+					</span>
+				</TableItem>
+				<TableItem className="orders__table-date">
+					{ moment( order.date_modified ).format( 'LLL' ) }
+				</TableItem>
+				<TableItem className="orders__table-status">
+					{ this.getOrderStatus( order.status ) }
+				</TableItem>
+				<TableItem className="orders__table-total">
+					{ order.total }
+				</TableItem>
+			</TableRow>
+		);
+	}
+
+	onPageClick = ( i ) => {
+		return () => {
+			this.props.setCurrentPage( this.props.siteId, i );
+		};
+	}
+
+	renderPageLink = ( i ) => {
+		// We want this to start at 1, not 0
+		i++;
+		return (
+			<li key={ i }>
+				{ ( i !== this.props.currentPage )
+					? <Button compact borderless onClick={ this.onPageClick( i ) }>{ i }</Button>
+					: <span>{ i }</span>
+				}
+			</li>
+		);
+	}
+
+	renderPagination = () => {
+		const { totalPages } = this.props;
+		if ( totalPages < 2 ) {
+			return null;
+		}
+
+		return (
+			<ul>
+				{ times( totalPages, this.renderPageLink ) }
+			</ul>
+		);
+	}
+
+	render() {
+		const { orders, ordersLoading, ordersLoaded, translate } = this.props;
+		const headers = (
+			<TableRow>
+				<TableItem isHeader>
+					<FormInputCheckbox aria-label={ translate( 'Select All', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **95** times:
`translate( 'Select all' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woocommerce-order-list)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.